packages/react-sdk/src/components/component-meta.ts

```diff
@@ -13,29 +13,21 @@ const WsComponentPropsMeta = z.object({
 
 export type WsComponentPropsMeta = z.infer<typeof WsComponentPropsMeta>;
 
-export type WsComponentMeta = {
-  /**
-   * body - can accept other components with dnd but not listed
-   * container - can accept other components with dnd
-   * control - usually form controls like inputs, without children
-   * embed - images, videos or other embeddable components, without children
-   * rich-text - editable text component
-   * rich-text-child - formatted text fragment, not listed in components list
-   */
-  type:
-    | "body"
-    | "container"
-    | "control"
-    | "embed"
-    | "rich-text"
-    | "rich-text-child";
-  label: string;
-  Icon: FunctionComponent<IconProps>;
-  presetStyle?: Style;
-  children?: Array<string>;
-};
+export const componentCategories = [
+  "general",
+  "typography",
+  "media",
+  "forms",
+] as const;
 
 const WsComponentMeta = z.object({
+  category: z.enum(componentCategories).optional(),
+  // body - can accept other components with dnd but not listed
```

Comment:
category removes the need for body to have separate type, can be removed in separate PR. Especially since we gonna have other not listed containers (fragment).

Reply:
good catch, done